packages/datadog-plugin-openai/src/index.js

```diff
@@ -113,42 +113,42 @@ class OpenApiPlugin extends TracingPlugin {
     }
 
     switch (methodName) {
-      case 'createFineTune':
+      case 'createFineTune': case 'fine_tuning.jobs.create': case 'fine-tune.create':
```

Comment:
Should probably not be putting multiple cases on the same line. I'm surprised the linter doesn't complain about this. 🤔

Reply:
ah fair, will make these each their own line. yeah, surprised this wasn't linted (I think I had done it this way while I was working to let myself know it was the same function but different name).

Reply:
You could use blank lines to group renames together.